atlas-service-parent/atlas-service/src/main/java/io/atlasmap/service/AtlasService.java

```diff
@@ -355,30 +351,29 @@ protected Response converterCheck(Mapping mapping, UriInfo uriInfo) {
                     Status.BAD_REQUEST);
         }
 
-        FieldType inputType = null;
-        FieldType outputType = null;
-
-        for (Field inputField : inputFields) {
-            if (inputField instanceof JavaField) {
-                inputType = ((JavaField) inputField).getFieldType();
-            }
-
-            for (Field outputField : outputFields) {
-                if (outputField instanceof JavaField) {
-                    outputType = ((JavaField) outputField).getFieldType();
-                }
-
-                Optional<AtlasConverter<?>> optionalConverter = conversionService.findMatchingConverter(inputType,
-                        outputType);
-                if (optionalConverter.isPresent()) {
-                    AtlasConverter<?> converter = optionalConverter.get();
-                    // TODO: return "ok"
-                } else {
-                    // TODO: return "Converter needed"
-                }
-
-            }
-        }
+//        FieldType inputType = null;
```

Comment:
overlooked this at first... definitely don't do this in any reason.

Reply:
Sorry, but I'm confused by what you mean by this. None of this code was being utilized for any valid purpose right now, and so I assume it just was only partially implemented and the author didn't have time to finish. Are you wanting me to delete it? I commented it out instead of deleting it because I wasn't sure what the original intention was and thought it best to save this skeleton code for when we might want to complete that implementation.

Reply:
don't change, keep it as is and file an issue

Reply:
So you ignored this and merged...

Reply:
Um, no, I did what you asked above.  You said don't change, so I didn't.

Reply:
Oh, crap, I just realized after hitting enter what you actually meant.  Again, I'm sorry, I certainly didn't mean to ignore you.  I now realize you meant don't change the original by commenting it out and file an issue (I guess about the unfinished code).  The latter I was going to do anyways, but sorry I misunderstood what you meant about the "don't change".

Reply:
I can just change it back if you like.

Reply:
Actually, I'd like to discuss this too. The main reason for commenting it out is to get rid of warnings in the editor.  That is important so that other warnings (e.g., about unused code) don't end up getting lost in a clutter of other warnings similar to this one that we'd end up just ignoring.